<ui version="4.0" >
 <author>Dag Andersen &lt;danders@get2net.dk></author>
 <class>KPlato::ConfigTaskPanelBase</class>
 <widget class="QWidget" name="KPlato::ConfigTaskPanelBase" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>419</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string/>
  </property>
  <layout class="QVBoxLayout" >
   <property name="margin" >
    <number>0</number>
   </property>
   <property name="spacing" >
    <number>6</number>
   </property>
   <item>
    <layout class="QHBoxLayout" >
     <property name="margin" >
      <number>0</number>
     </property>
     <property name="spacing" >
      <number>6</number>
     </property>
     <item>
      <widget class="QLabel" name="leaderlabel" >
       <property name="whatsThis" >
        <string>The person responsible for this task.

This is not limited to persons available in a resource group but can be anyone. You can even directly access your address book with the Choose button.</string>
       </property>
       <property name="text" >
        <string>Responsible:</string>
       </property>
       <property name="wordWrap" >
        <bool>false</bool>
       </property>
       <property name="buddy" >
        <cstring>leaderfield</cstring>
       </property>
      </widget>
     </item>
     <item>
      <widget class="KLineEdit" native="1" name="leaderfield" >
       <property name="sizePolicy" >
        <sizepolicy>
         <hsizetype>5</hsizetype>
         <vsizetype>5</vsizetype>
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="whatsThis" >
        <string>The person responsible for this task.

This is not limited to persons available in a resource group but can be anyone. You can even directly access your address book with the Choose button.</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="chooseLeader" >
       <property name="toolTip" >
        <string>Insert a person from your address book</string>
       </property>
       <property name="whatsThis" >
        <string>Insert a person from your address book.</string>
       </property>
       <property name="text" >
        <string>Choose...</string>
       </property>
       <property name="shortcut" >
        <string/>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QGroupBox" name="schedulingGroup" >
     <property name="title" >
      <string>Timing</string>
     </property>
     <layout class="QGridLayout" >
      <property name="margin" >
       <number>9</number>
      </property>
      <property name="spacing" >
       <number>6</number>
      </property>
      <item row="0" column="0" >
       <widget class="QLabel" name="textLabel3" >
        <property name="sizePolicy" >
         <sizepolicy>
          <hsizetype>5</hsizetype>
          <vsizetype>5</vsizetype>
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text" >
         <string>Schedule:</string>
        </property>
        <property name="wordWrap" >
         <bool>false</bool>
        </property>
        <property name="buddy" >
         <cstring>scheduleType</cstring>
        </property>
       </widget>
      </item>
      <item row="0" column="1" colspan="2" >
       <widget class="KComboBox" name="scheduleType" >
        <property name="sizePolicy" >
         <sizepolicy>
          <hsizetype>5</hsizetype>
          <vsizetype>0</vsizetype>
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <item>
         <property name="text" >
          <string>As Soon as Possible</string>
         </property>
        </item>
        <item>
         <property name="text" >
          <string>As Late as Possible</string>
         </property>
        </item>
        <item>
         <property name="text" >
          <string>Must Start On</string>
         </property>
        </item>
        <item>
         <property name="text" >
          <string>Must Finish On</string>
         </property>
        </item>
        <item>
         <property name="text" >
          <string>Start Not Earlier Than</string>
         </property>
        </item>
        <item>
         <property name="text" >
          <string>Finish Not Later Than</string>
         </property>
        </item>
        <item>
         <property name="text" >
          <string>Fixed Interval</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="0" column="3" >
       <layout class="QHBoxLayout" >
        <property name="margin" >
         <number>0</number>
        </property>
        <property name="spacing" >
         <number>6</number>
        </property>
        <item>
         <widget class="KDateWidget" native="1" name="scheduleStartDate" >
          <property name="sizePolicy" >
           <sizepolicy>
            <hsizetype>1</hsizetype>
            <vsizetype>0</vsizetype>
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="focusPolicy" >
           <enum>Qt::WheelFocus</enum>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QTimeEdit" name="scheduleStartTime" />
        </item>
       </layout>
      </item>
      <item row="1" column="3" >
       <layout class="QHBoxLayout" >
        <property name="margin" >
         <number>0</number>
        </property>
        <property name="spacing" >
         <number>6</number>
        </property>
        <item>
         <widget class="KDateWidget" native="1" name="scheduleEndDate" >
          <property name="sizePolicy" >
           <sizepolicy>
            <hsizetype>1</hsizetype>
            <vsizetype>0</vsizetype>
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="focusPolicy" >
           <enum>Qt::WheelFocus</enum>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QTimeEdit" name="scheduleEndTime" />
        </item>
       </layout>
      </item>
      <item row="2" column="0" >
       <widget class="QLabel" name="textLabel3_2" >
        <property name="sizePolicy" >
         <sizepolicy>
          <hsizetype>5</hsizetype>
          <vsizetype>5</vsizetype>
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text" >
         <string>Estimate:</string>
        </property>
        <property name="wordWrap" >
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item row="2" column="1" >
       <widget class="KComboBox" name="estimateType" >
        <property name="sizePolicy" >
         <sizepolicy>
          <hsizetype>5</hsizetype>
          <vsizetype>0</vsizetype>
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="editable" >
         <bool>false</bool>
        </property>
        <item>
         <property name="text" >
          <string>Effort</string>
         </property>
        </item>
        <item>
         <property name="text" >
          <string>Duration</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="2" column="2" colspan="2" >
       <widget class="QWidget" native="1" name="durationHolder" >
        <property name="minimumSize" >
         <size>
          <width>250</width>
          <height>18</height>
         </size>
        </property>
       </widget>
      </item>
      <item row="3" column="2" colspan="2" >
       <layout class="QHBoxLayout" >
        <property name="margin" >
         <number>0</number>
        </property>
        <property name="spacing" >
         <number>6</number>
        </property>
        <item>
         <widget class="QLabel" name="textLabel1_2" >
          <property name="sizePolicy" >
           <sizepolicy>
            <hsizetype>5</hsizetype>
            <vsizetype>5</vsizetype>
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text" >
           <string>Optimistic:</string>
          </property>
          <property name="wordWrap" >
           <bool>false</bool>
          </property>
          <property name="buddy" >
           <cstring>optimisticValueoptimisticValue</cstring>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSpinBox" name="optimisticValue" >
          <property name="suffix" >
           <string>%</string>
          </property>
          <property name="maximum" >
           <number>0</number>
          </property>
          <property name="minimum" >
           <number>-99</number>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="textLabel2_2" >
          <property name="sizePolicy" >
           <sizepolicy>
            <hsizetype>5</hsizetype>
            <vsizetype>5</vsizetype>
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text" >
           <string>Pessimistic:</string>
          </property>
          <property name="alignment" >
           <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
          </property>
          <property name="wordWrap" >
           <bool>false</bool>
          </property>
          <property name="buddy" >
           <cstring>pessimisticValue</cstring>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSpinBox" name="pessimisticValue" >
          <property name="suffix" >
           <string>%</string>
          </property>
          <property name="maximum" >
           <number>999</number>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="descriptionlabell6" >
     <property name="text" >
      <string>Note:</string>
     </property>
     <property name="wordWrap" >
      <bool>false</bool>
     </property>
     <property name="buddy" >
      <cstring>descriptionfield</cstring>
     </property>
    </widget>
   </item>
   <item>
    <widget class="KTextEdit" native="1" name="descriptionfield" />
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11" />
 <customwidgets>
  <customwidget>
   <class>KTextEdit</class>
   <extends>QWidget</extends>
   <header>ktextedit.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>KDateWidget</class>
   <extends>QWidget</extends>
   <header>kdatewidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>KLineEdit</class>
   <extends>QWidget</extends>
   <header>klineedit.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>KComboBox</class>
   <extends>QComboBox</extends>
   <header>kcombobox.h</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>leaderfield</tabstop>
  <tabstop>chooseLeader</tabstop>
  <tabstop>scheduleType</tabstop>
  <tabstop>scheduleStartDate</tabstop>
  <tabstop>scheduleEndDate</tabstop>
  <tabstop>estimateType</tabstop>
  <tabstop>optimisticValue</tabstop>
  <tabstop>pessimisticValue</tabstop>
  <tabstop>descriptionfield</tabstop>
 </tabstops>
 <includes>
  <include location="local" >klineedit.h</include>
  <include location="local" >kcombobox.h</include>
  <include location="local" >kcombobox.h</include>
  <include location="local" >kdatewidget.h</include>
  <include location="local" >kdatewidget.h</include>
  <include location="local" >kptdurationwidget.h</include>
  <include location="local" >ktextedit.h</include>
 </includes>
 <resources/>
 <connections/>
</ui>
